components/camera/camera.go

```diff
@@ -189,6 +189,7 @@ func NewFromSource(
 	}
 	actualSystem := syst
 	if actualSystem == nil {
+		//nolint:staticcheck
```

Comment:
staticcheck doesn't handle the generics in gostream so believes this is a mismatched type